From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Stultz <john.stultz@linaro.org>
Date: Thu, 13 Feb 2020 22:52:08 +0000
Subject: ANDROID: Kconfig.gki: Remove most of the built in qcom clks

There was a desire to see COMMON_CLK_QCOM set as a module
instead of building it in. So this patch removes it and
a few dependent modules from the built in list, and adds
them to the db845c_gki.fragment file as modules.

Bug: 149414941
Change-Id: I6fecab5f529c5db1d7788e330ce8815dc22c4d1f
Signed-off-by: John Stultz <john.stultz@linaro.org>
Signed-off-by: Saravana Kannan <saravanak@google.com>
[joneslee: Removed init/Kconfig.gki elements]
Signed-off-by: Lee Jones <lee.jones@linaro.org>
---
 arch/arm64/configs/db845c_gki.fragment | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/configs/db845c_gki.fragment b/arch/arm64/configs/db845c_gki.fragment
--- a/arch/arm64/configs/db845c_gki.fragment
+++ b/arch/arm64/configs/db845c_gki.fragment
@@ -40,7 +40,10 @@ CONFIG_USB_ROLE_SWITCH=m
 CONFIG_USB_ULPI_BUS=m
 CONFIG_MMC_SDHCI_MSM=m
 CONFIG_RTC_DRV_PM8XXX=m
+CONFIG_COMMON_CLK_QCOM=m
 CONFIG_COMMON_CLK_SCMI=m
+CONFIG_SDM_GPUCC_845=m
+CONFIG_QCOM_CLK_RPMH=m
 CONFIG_MSM_GCC_8998=m
 CONFIG_SDM_DISPCC_845=m
 CONFIG_HWSPINLOCK_QCOM=m
